lib/phlex/kit.rb

```diff
@@ -52,6 +52,8 @@ def const_added(name)
 		if Class === constant && constant < Phlex::SGML
 			constant.include(self)
 
+			constant = nil
```

Comment:
We need the old constant to be GC’d here and not referenced by the closures below.